charts/osschallenge/values.yaml

```diff
@@ -0,0 +1,108 @@
+    initialDelaySeconds: 60
+    # osschallenge.livenessProbe.periodSeconds -- How often (in seconds) to perform the probe
+    periodSeconds: 10
+    # osschallenge.livenessProbe.timeoutSeconds -- Number of seconds after which the probe times out
+    timeoutSeconds: 5
+    # osschallenge.livenessProbe.failureThreshold -- Number of tries to perform the probe
+    failureThreshold: 6
+    # osschallenge.livenessProbe.successThreshold -- Minimum consecutive successes for the probe to be considered successful after having failed
+    successThreshold: 1
+  readinessProbe:
+    # osschallenge.readinessProbe.enabled -- Enable readiness probe on osschallenge
+    enabled: true
+    # osschallenge.readinessProbe.initialDelaySeconds -- Number of seconds after the container has started before readiness probe is initiated
+    initialDelaySeconds: 30
+    # osschallenge.readinessProbe.periodSeconds -- How often (in seconds) to perform the probe
+    periodSeconds: 10
+    # osschallenge.readinessProbe.timeoutSeconds -- Number of seconds after which the probe times out
+    timeoutSeconds: 5
+    # osschallenge.readinessProbe.failureThreshold -- Number of tries to perform the probe
+    failureThreshold: 6
+    # osschallenge.readinessProbe.successThreshold -- Minimum consecutive successes for the probe to be considered successful after having failed
+    successThreshold: 1
+  settings:
+    # django settings for osschallenge
+    # osschallenge.settings.emailHost -- Connection string of SMTP server to send mails
+    emailHost: smtp.example.com
+    # osschallenge.settings.emailPort -- Port for the SMTP server
+    emailPort: 587
+    # osschallenge.settings.emailFrom -- Default email address to use for various responses
+    emailFrom: osschallenge@oss-challenge.ch
+    # osschallenge.settings.emailUser -- Username for the SMTP server
+    emailUser: example@example.com
+    # osschallenge.settings.emailPassword -- Password for the SMTP server
+    emailPassword: ~
+    # osschallenge.settings.siteUrl -- URL for OSS-Challenge
+    siteUrl: oss-chalenge.ch
+
+ingress:
+  # ingress.enabled -- Enable ingress for osschallenge
+  enabled: false
+  # ingress.annotations -- Ingress annotations
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  # ingress.hosts -- Ingress hostnames
+  hosts: ["localhost"]
+  #  - osschallenge-test.example.com
+  # ingress.tls -- Ingress TLS options
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - osschallenge-test.example.com
+
+postgresql:
+  # postgresql.enabled -- Enable PostgreSQL for persistence
+  enabled: true
+  # postgresql.postgresqlDatabase -- PostgreSQL database name
+  postgresqlDatabase: osschallenge
+  # postgresql.postgresqlUsername -- PostgreSQL user name
+  postgresqlUsername: osschallenge
+  # postgresql.postgresqlPassword -- Set a password for PostgreSQL
+  postgresqlPassword: ~
```

Comment:
I can't find where this is used. Shouldn't it be in the Secret as well?

Reply:
found the `{{- include "osschallenge.djangoDatabasePassword" . | nindent 12 }}` lines